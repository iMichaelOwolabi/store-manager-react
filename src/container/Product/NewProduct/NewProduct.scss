@import '../../../partial-styles/fonts.scss';

.create-product {
  text-align: center;
  margin: {
    top: 2.5rem;
    right: auto;
    left: auto;
  }
  width: 50%;
  min-width: 300px;
}
.create-product input {
  width: 90%;
  padding: .7rem;
  outline: none;
  font-weight: 100;
}
.create-product p {
  font-weight: 200;
  padding-top: 1rem;
}
.upload {
  background-color: #888;
  border-radius: 5px;
}
.create-product button {
  padding: 10px;
  border-radius: .3rem;
  font-size: 1.1rem;
  min-width: 10%;
  font-weight: 500;
  margin-top: 15px;
  outline: none;
}