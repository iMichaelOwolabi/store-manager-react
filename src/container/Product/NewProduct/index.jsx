import React, { Fragment } from 'react';
import AuthNav from '../../../components/AuthNav';

const NewProduct = () => (
  <Fragment>
    <AuthNav />
    <div className="container">
      New Product
    </div>
  </Fragment>
);
export default NewProduct;
