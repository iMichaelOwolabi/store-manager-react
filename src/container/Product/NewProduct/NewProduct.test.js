import React from 'react';
import { shallow } from 'enzyme';
import NewProduct from '.';

describe('Test the 404 Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<NewProduct />);
    expect(wrapper).toMatchSnapshot();
  });
});
