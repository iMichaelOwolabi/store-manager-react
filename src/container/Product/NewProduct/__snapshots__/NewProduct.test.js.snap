// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the 404 Component should match snapshot 1`] = `
<Fragment>
  <AuthNav />
  <div
    className="container"
  >
    New Product
  </div>
</Fragment>
`;
